### ISSUE TITLE
Enable pheno level session for response and query

### ISSUE THREAD

**Issue by alyssadai** [neurobagel/query-tool#78]
This issue is to address two things:

- [x] The query tool can send requests for # of pheno and imaging sessions according to the new query variables:

https://github.com/neurobagel/federation-api/blob/771b1a2fd21030ee3a6fe1a8bc6c37db401a8fd1/app/api/models.py#L20-L21

- [x] Query tool can understand the new response model from the API:

https://github.com/neurobagel/api/blob/c5c6222f77995e1b723e5ac24f7041151556b458/app/api/models.py#L13-L24

Steps:

- [x] update query UI for # of imaging sessions
- [x] add query UI for # of pheno sessions
- [x] add tests to check that this is correctly mapped to new query parameters
- [x] add test to ensure the tool creates the correct participant.tsv files when given the new API response with `session_type`:

```json
[
  {
    "dataset_uuid": "http://neurobagel.org/vocab/e0c7d08c-edcc-4c2a-816a-306878ed7be2",
    "dataset_name": "BIDS synthetic",
    "dataset_portal_uri": "https://github.com/bids-standard/bids-examples",
    "dataset_total_subjects": 5,
    "records_protected": false,
    "num_matching_subjects": 5,
    "subject_data": [
      {
        "sub_id": "sub-01",
        "session_id": "ses-01",
        "num_matching_phenotypic_sessions": 2,
        "num_matching_imaging_sessions": 2,
        "session_type": "http://neurobagel.org/vocab/ImagingSession",
        "age": null,
        "sex": null,
        "diagnosis": [
          null
        ],
        "subject_group": null,
        "assessment": [
          null
        ],
        "image_modal": [
          "http://purl.org/nidash/nidm#T1Weighted",
          "http://purl.org/nidash/nidm#FlowWeighted"
        ],
        "session_file_path": "/data/neurobagel/bagel-cli/bids-examples/synthetic/sub-01/ses-01"
      },
      {
        "sub_id": "sub-01",
        "session_id": "ses-01",
        "num_matching_phenotypic_sessions": 2,
        "num_matching_imaging_sessions": 2,
        "session_type": "http://neurobagel.org/vocab/PhenotypicSession",
        "age": 34.1,
        "sex": "http://purl.bioontology.org/ontology/SNOMEDCT/248152002",
        "diagnosis": [
          null
        ],
        "subject_group": "http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C94342",
        "assessment": [
          "https://www.cognitiveatlas.org/task/id/trm_57964b8a66aed",
          "https://www.cognitiveatlas.org/task/id/tsk_4a57abb949ece"
        ],
        "image_modal": [
          null
        ],
        "session_file_path": null
      },
      ...
]
```

- [x] add capacity to receive response with new `session_type` parameter  / make that test pass

**Comment by rmanaem** [comment]
Unblocked since https://github.com/neurobagel/planning/issues/96 has been addressed

**Comment by surchs** [comment]
@alyssadai do you have an example response from the API that includes `session_type`? mainly for testing purposes

**Comment by alyssadai** [comment]
Here you go @surchs:

```json
[
  {
    "dataset_uuid": "http://neurobagel.org/vocab/e0c7d08c-edcc-4c2a-816a-306878ed7be2",
    "dataset_name": "BIDS synthetic",
    "dataset_portal_uri": "https://github.com/bids-standard/bids-examples",
    "dataset_total_subjects": 5,
    "records_protected": false,
    "num_matching_subjects": 5,
    "subject_data": [
      {
        "sub_id": "sub-01",
        "session_id": "ses-01",
        "num_matching_phenotypic_sessions": 2,
        "num_matching_imaging_sessions": 2,
        "session_type": "http://neurobagel.org/vocab/ImagingSession",
        "age": null,
        "sex": null,
        "diagnosis": [
          null
        ],
        "subject_group": null,
        "assessment": [
          null
        ],
        "image_modal": [
          "http://purl.org/nidash/nidm#T1Weighted",
          "http://purl.org/nidash/nidm#FlowWeighted"
        ],
        "session_file_path": "/data/neurobagel/bagel-cli/bids-examples/synthetic/sub-01/ses-01"
      },
      {
        "sub_id": "sub-01",
        "session_id": "ses-01",
        "num_matching_phenotypic_sessions": 2,
        "num_matching_imaging_sessions": 2,
        "session_type": "http://neurobagel.org/vocab/PhenotypicSession",
        "age": 34.1,
        "sex": "http://purl.bioontology.org/ontology/SNOMEDCT/248152002",
        "diagnosis": [
          null
        ],
        "subject_group": "http://ncicb.nci.nih.gov/xml/owl/EVS/Thesaurus.owl#C94342",
        "assessment": [
          "https://www.cognitiveatlas.org/task/id/trm_57964b8a66aed",
          "https://www.cognitiveatlas.org/task/id/tsk_4a57abb949ece"
        ],
        "image_modal": [
          null
        ],
        "session_file_path": null
      },
      ...
]
```

**Comment by surchs** [comment]
Thanks a lot @alyssadai !

**Comment by alyssadai** [comment]
Blocked until #97 is reviewed to avoid messy rebasing

**Comment by rmanaem** [comment]
| DatasetID                                                        | SubjectID | SessionID | SessionType                                   | Age  | Sex                                                     | Diagnosis | Assessment                                                                                                           | SessionFilePath                                                      | NumPhenotypicSessions | NumImagingSessions | Modality                                                                           |
|------------------------------------------------------------------|-----------|-----------|-----------------------------------------------|------|---------------------------------------------------------|-----------|----------------------------------------------------------------------------------------------------------------------|------------------------------------------------------------------|-----------------------|--------------------|------------------------------------------------------------------------------------|
| http://neurobagel.org/vocab/e0c7d08c-edcc-4c2a-816a-306878ed7be2 | sub-01    | ses-01    | http://neurobagel.org/vocab/ImagingSession    | null | null                                                    | null      | null                                                                                                                 | /data/neurobagel/bagel-cli/bids-examples/synthetic/sub-01/ses-01 | 1                     | 1                  | [http://purl.org/nidash/nidm#T1Weighted, http://purl.org/nidash/nidm#FlowWeighted] |
| http://neurobagel.org/vocab/e0c7d08c-edcc-4c2a-816a-306878ed7be2 | sub-01    | ses-01    | http://neurobagel.org/vocab/PhenotypicSession | 34.1 | http://purl.bioontology.org/ontology/SNOMEDCT/248152002 | null      | [https://www.cognitiveatlas.org/task/id/trm_57964b8a66aed, https://www.cognitiveatlas.org/task/id/tsk_4a57abb949ece] | null                                                             | 1                     | 1                  | null                                                                               |

**Comment by surchs** [comment]
:rocket: Issue was released in [`v0.2.0`](https://github.com/neurobagel/query-tool/releases/tag/v0.2.0) :rocket:
